From a40d101234021f51d736be165aff66b1fa82a39a Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Thu, 2 Aug 2012 23:17:43 +0800
Subject: ENGR00219178 [MX6]Need to avoid using GPIO_1 on latest B4 sabresd
 board

For the latest B4 sabresd board, GPIO_1 is connected to PRO_B, we
use this pin as battery charge_done led control, need to avoid using
this pin as GPIO for B4 board, or system will reboot when this pin's
level is changed.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index ae8a613..6116d5b 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -1499,8 +1499,14 @@ static void gps_power_on(bool on)
 static struct gpio_led imx6q_gpio_leds[] = {
 	GPIO_LED(SABRESD_CHARGE_NOW, "chg_now_led", 0, 1,
 		"charger-charging"),
+/* For the latest B4 board, this GPIO_1 is connected to POR_B,
+which will reset the whole board if this pin's level is changed,
+so, for the latest board, we have to avoid using this pin as
+GPIO. */
+#if 0
 	GPIO_LED(SABRESD_CHARGE_DONE, "chg_done_led", 0, 1,
 			"charger-full"),
+#endif
 };
 
 static struct gpio_led_platform_data imx6q_gpio_leds_data = {
-- 
1.7.9.5

